src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/resource/PrisonResource.kt

```diff
@@ -139,8 +139,9 @@ class PrisonResource(private val prisonService: PrisonService, private val addre
   )
   fun getPrisonNames(
     @Parameter(description = "If active is not set, return all prisons, otherwise return only the active or inactive ones based on the value", example = "true", required = false) @RequestParam active: Boolean?,
+    @Parameter(description = "If name is not set, return all prisons, otherwise return only the name either is active or inactive ones based on the  prison name value given", example = "WDI", required = false) @RequestParam name: String?,
```

Comment:
this description is not clear
what about 
> Return the names of all prisons if the parameter is not set, otherwise return only the one corresponding to the code. The filtering on active still applies.